### PR TITLE
Add ostree support

### PR DESCRIPTION
**Pull request by jkozol**
This PR is a MVP. For some image types the details page will be empty but instead of changing the routing I am leaving it empty since soon other blueprint customization will be added to the wizard and the routing would need to be changed back.

Also see issue https://github.com/osbuild/cockpit-composer/issues/1465

### CODE REVIEW

**Comment by teg** [comment]
Maybe not relevant for the first version, but might make sense to look at how these things are done in edge fleet managment in crc if we can steal anything from there.

File: components/Wizard/formComponents/FormGroupCustom.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    return { validated: "error" };
+  }
+
+  if ((touched || validateOnMount) && submitError) {
+    return { validated: "error" };
+  }
+
+  if ((touched || validateOnMount) && warning) {
+    return { validated: "warning" };
+  }
```
Comment (lucasgarfield) [inline components/Wizard/formComponents/FormGroupCustom.js:16]: I can't find `meta.warning` in the react-final-form docs, just double checking that it really exists.

https://final-form.org/docs/react-final-form/types/FieldRenderProps

Comment (jkozol) [reply components/Wizard/formComponents/FormGroupCustom.js:16]: https://data-driven-forms.org/schema/validator-warnings
https://www.patternfly.org/v4/components/text-input#textinput

So the warning in meta should mimic the warning option in Patternfly's form validation

**Comment by jkozol** [comment]
> Maybe not relevant for the first version, but might make sense to look at how these things are done in edge fleet managment in crc if we can steal anything from there.

@teg Unfortunately they don't allow customizations and only build `RHEL for Edge Commit (.tar)` and `RHEL for Edge Installer (.iso)` image types.

I do want to steal their review page though :) I think it is very clean.

File: components/Wizard/formComponents/FormGroupCustom.js (hunk 1 above)
Comment (jkozol) [reply components/Wizard/formComponents/FormGroupCustom.js:16]: Currently this isn't used by us but will probably be useful in the future. I.e. if a user sets a strange value that won't necessarily fail but might. Image size of 10 TB or something...

**Comment by lucasgarfield** [comment]
Everything looks good to me, I think we can merge!

**Review by lucasgarfield — APPROVED** [review]

**Pull request merged**